blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScm.java

```diff
@@ -230,7 +227,7 @@ public HttpResponse validateAndCreate(@JsonBody JSONObject request) {
             //Now we know the token is valid. Lets find credential
             StandardUsernamePasswordCredentials githubCredential = CredentialsUtils.findCredential(getId(), StandardUsernamePasswordCredentials.class, new BlueOceanDomainRequirement());
 
-            final StandardUsernamePasswordCredentials credential = new UsernamePasswordCredentialsImpl(CredentialsScope.USER, "github", "Github Access Token", authenticatedUser.getId(), accessToken);
+            final StandardUsernamePasswordCredentials credential = new UsernamePasswordCredentialsImpl(CredentialsScope.USER, getId(), "Github Access Token", authenticatedUser.getId(), accessToken);
```

Comment:
This is intended to avoid two credentials with the same ID of "github" - when I originally tested this code, the cred tied to GitHub Enterprise was actually greyed out in the UI so I assumed there was some kind of conflict happening. This also led to the `credentialId` field in the response being null which is a problem.